blogposts/2021/workspaces-of-sourcegraph.md

```diff
@@ -0,0 +1,187 @@
+---
+title: The workspaces of Sourcegraph
+author: Rob Rhyne
+authorUrl: https://twitter.com/robrhyne
+publishDate: 2021-02-10T18:00+02:00
+tags: [blog, remote]
+slug: workspaces-of-sourcegraph
+published: true
+---
+
+This started as a design research thread in Slack and escalated into something awesome very quickly! Here are all the workspaces and dev setups of our team.
```

Comment:
@rrhyne can you add some context to the introduction? It would be interesting to know what you are researching that prompted the question and/or why a great workspace is so important.